static/docs/commands-reference/install.md

```diff
@@ -23,13 +50,204 @@ Install dvc hooks into the repository
+files, and now `dvc status` tells us the data files match the DVC files.
+
+Now let's see how this changes after running `dvc install`
+
+```dvc
+    $ git checkout master
+    Previous HEAD position was d13ba9a add featurization stage
+    Switched to branch 'master'
+    Your branch is up to date with 'origin/master'.
+
+    $ dvc checkout
+    [##############################] 100% Checkout finished!
+```
+
+Start by resetting the workspace to he at the head commit.
+
 ```dvc
     $ dvc install
+
     $ cat .git/hooks/pre-commit
-      #!/bin/sh
-      exec dvc status
+    #!/bin/sh
+    exec dvc status
+    
     $ cat .git/hooks/post-checkout
-      #!/bin/sh
-      exec dvc checkout
-    $ git checkout mybranch # will call `dvc checkout` automatically
+    #!/bin/sh
+    exec dvc checkout
 ```
+
+The two Git hooks have been installed, and the one of interest for this exercise
+is the `post-checkout` script which runs after `git checkout`. 
+
+```dvc
+    $ git checkout 6-featurization
+    Note: checking out '6-featurization'.
+
+    You are in 'detached HEAD' state. ...
+
+    HEAD is now at d13ba9a add featurization stage
+    [##############################] 100% Checkout finished!
+
+    $ dvc status
+
+    Pipeline is up to date. Nothing to reproduce.
+```
+
+Look carefully at this output and it is clear that the `dvc checkout` command
+has indeed been run. As a result the workspace is up-to-date with the data files
+matching what is referenced by the DVC files.
+
+## Example: Showing DVC status on Git commit
+
+The other hook installed by `dvc install` runs before `git commit` operation.
+To see see what that does, start with the same workspace, making sure it is
+not in the detached HEAD state from the previous example.
+
+If we simply edit one of the code files:
+
+```dvc
+    $ vi src/featurization.py 
+
+    $ git commit -a -m 'modified featurization'
+
+    featurize.dvc:
+        changed deps:
+            modified:           src/featurization.py
+    [master 1116ddc] modified featurization
+     1 file changed, 1 insertion(+), 1 deletion(-)
+```
+
+We see that `dvc status` output has appeared in the `git commit` interaction.
+This new behavior corresponds to the Git hook which was installed, and it
+helpfully informs us the workspace is out of sync.  We should therefore run
+the `dvc repro` command.
+
+```dvc
+    $ dvc repro evaluate.dvc 
+
+    ... much output
+    To track the changes with git run:
+
+        git add featurize.dvc train.dvc evaluate.dvc
+
+    $ git status -s
+     M auc.metric
+     M evaluate.dvc
+     M featurize.dvc
+     M src/featurization.py
+     M train.dvc
+
+    $ git commit -a -m 'updated data after modified featurization'
+    Pipeline is up to date. Nothing to reproduce.
+    [master 78d0c44] modified featurization
+     5 files changed, 12 insertions(+), 12 deletions(-)
+
+```
+
+After rerunning the DVC pipeline, of course the data files are in sync with
```

Comment:
copy and hight the phrase `Pipeline is up to date. Nothing to reproduce.` once again here? So that user can easily find in the code snippet what is happening.

Reply:
I think you mean to insert some blank lines making it easier to find the text from `dvc status`?

Reply:
that one is good, but I would also mention the phrase in the paragraph itself